### DISCUSSION TITLE
Correct Connection?

### DISCUSSION THREAD

**Discussion by deepbluemind74** [sekigon-gonnoc/Pico-PIO-USB#7] [General]
According to the description - would that be the correct connection?

![pico-add-usb](https://user-images.githubusercontent.com/58854618/154962083-afc60cd1-76be-4897-b186-4b2fe44dbc7a.png)

**Comment by sekigon-gonnoc** [comment]
In the default settings, yes.

**Reply by deepbluemind74** [reply]
So no need for resitors?

**Reply by sekigon-gonnoc** [reply]
Around 27ohm is recommended but not necessary

**Reply by deepbluemind74** [reply]
Where do I put this?

**Reply by mingpepe** [reply]
Like this
![image](https://user-images.githubusercontent.com/6778493/179219717-658810cb-88eb-433b-b764-3cb2d5a90689.png)

https://datasheets.raspberrypi.com/rp2040/hardware-design-with-rp2040.pdf
